### DISCUSSION TITLE
ESP32 question

### DISCUSSION THREAD

**Discussion by Firemedic04** [forkineye/ESPixelStick#423] [General]
Hey guys, I have been testing with the LOLIN32 1.0 board found [HERE](https://www.amazon.com/Development-Bluetooth-Module-Lolin32-Arduino/dp/B07RZ8HHP9/ref=sr_1_3?keywords=lolin32&qid=1637972909&qsid=134-3278423-9858567&sr=8-3&sres=B082HSZWJN%2CB07RZ8HHP9%2CB08NLP5W2D%2CB07MK1WF91%2CB07QDFP3WC%2CB07HG6WT5V%2CB07F77NFGW%2CB07L9HCJ2V%2CB08FBK2DK5%2CB085L5L7B1%2CB0829RYP8V%2CB077KJNVFP%2CB07YXH57B8%2CB097PSJRB7%2CB07CL4GLQW%2CB073JP6WK4%2CB01MTGBKPQ%2CB07B7JJQMF%2CB0856PHNLX%2CB00026V7Z4)  Is the program usable on this board or does it need to be a PRO board?  Other than 16MB Flash is there another big difference?

**Comment by forkineye** [comment]
The Pro model has PSRAM on it. You can use the version you linked, but you'll need to use the Generic ESP32 4MB binary or d1_mini32 environment if building yourself.
